ipa-core/src/net/test.rs

```diff
@@ -318,17 +508,40 @@ impl TestServerBuilder {
     }
 }
 
-fn get_test_certificate_and_key(id: HelperIdentity) -> (&'static [u8], &'static [u8]) {
+pub struct ClientIdentities {
+    pub helper: ClientIdentity<Helper>,
+    pub shard: ClientIdentity<Shard>,
+}
+
+#[must_use]
+pub fn create_ids(disable_https: bool, id: HelperIdentity, ix: ShardIndex) -> ClientIdentities {
```

Comment:
should it be a constructor on `ClientIdentities`?

Reply:
Sure. The conditional on disable_https had me on the fence